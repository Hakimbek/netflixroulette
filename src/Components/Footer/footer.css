.footer {
  margin-top: auto;
  display: flex;
  justify-content: center;
  padding: 10px;
  background-color: #424242;
}
