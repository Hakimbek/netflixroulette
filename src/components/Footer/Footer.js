import React from "react";
import PropTypes from "prop-types";
import styles from "./footer.module.css";

function Footer(props) {
  return <div className={styles.footer_wrapper}>{props.children}</div>;
}

Footer.propTypes = {
  children: PropTypes.element.isRequired,
};

export default Footer;
