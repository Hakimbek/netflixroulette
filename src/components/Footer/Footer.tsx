import Logo from "../common/Logo/Logo";

import styles from "./footer.module.css";

function Footer() {
  return (
    <div className={styles.footer}>
      <Logo />
    </div>
  );
}

export default Footer;
