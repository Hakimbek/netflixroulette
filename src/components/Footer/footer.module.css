.footer_wrapper {
  display: flex;
  justify-content: center;
  padding: 10px;
  background-color: #424242;
}
