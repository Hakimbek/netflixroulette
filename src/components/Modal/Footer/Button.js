import styled from "styled-components";

const Button = styled.button`
  width: 200px;
  height: 50px;
  margin: 10px;
  border: 1px solid ${(props) => props.borderColor};
  border-radius: 5px;
  background-color: ${(props) => props.backgroundColor};
  color: ${(props) => props.color};
  font-size: 20px;
  cursor: pointer;
`;

export default Button;
