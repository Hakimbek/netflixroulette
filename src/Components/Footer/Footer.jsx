import PropTypes from "prop-types";
import "./footer.css";

function Footer(props) {
  return <div className="footer">{props.children}</div>;
}

Footer.propTypes = {
  children: PropTypes.element,
};

export default Footer;
