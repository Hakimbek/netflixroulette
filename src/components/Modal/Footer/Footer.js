import React from "react";
import Button from "./Button";
import styles from "../modal.module.css";

function Footer() {
  return (
    <div className={styles.footer}>
      <Button
        color={"#F65261"}
        backgroundColor={"#232323"}
        borderColor={"#F65261"}
      >
        RESET
      </Button>
      <Button
        color={"white"}
        backgroundColor={"#F65261"}
        borderColor={"#F65261"}
      >
        SUBMIT
      </Button>
    </div>
  );
}

export default Footer;
