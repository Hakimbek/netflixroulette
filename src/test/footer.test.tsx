import { render } from "@testing-library/react";
import Footer from "../components/Footer/Footer";

describe("Footer", () => {
  it("renders Footer component", () => {
    const { asFragment } = render(<Footer />);
    expect(asFragment()).toMatchSnapshot();
  });
});
