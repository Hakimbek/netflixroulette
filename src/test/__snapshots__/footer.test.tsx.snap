// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer render Footer component 1`] = `
<DocumentFragment>
  <div
    class="footer"
  >
    <p
      class="logo"
    >
      <b>
        netflix
      </b>
      roulette
    </p>
  </div>
</DocumentFragment>
`;

exports[`Footer renders Footer component 1`] = `
<DocumentFragment>
  <div
    class="footer"
  >
    <p
      class="logo"
    >
      <b>
        netflix
      </b>
      roulette
    </p>
  </div>
</DocumentFragment>
`;
